db/fixtures/miq_event_definitions.csv

```diff
@@ -51,6 +51,17 @@ host_auth_invalid,Host Auth Invalid,Default,auth_validation
 host_auth_unreachable,Host Auth Unreachable,Default,auth_validation
 host_auth_incomplete,Host Auth Incomplete Credentials,Default,auth_validation
 host_auth_error,Host Auth Error,Default,auth_validation
+request_host_reset,Host Reset Request,Default,host_operations
+request_host_start,Host Start Request,Default,host_operations
+request_host_stop,Host Stop Request,Default,host_operations
+request_host_standby,Host Standby Request,Default,host_operations
+request_host_enter_maintenance_mode,Host Maintenance Enter Request,Default,host_operations
+request_host_exit_maintenance_mode,Host Maintenance Exit Request,Default,host_operations
+request_host_shutdown,Host Shutdown Request,Default,host_operations
+request_host_reboot,Host Reboot Request,Default,host_operations
+request_host_enable_vmotion,Host Vmotion Enable Request,Default,host_operations
+request_host_disable_vmotion,Host Vmotion Disable Request,Default,host_operations
+request_host_vmotion_enabled,Host Vmotion Enabled Request,Default,host_operations
```

Comment:
These all make sense expect for `request_host_vmotion_enabled`.  This is called from the `vmotion_enabled?`  [host.rb:488](https://github.com/ManageIQ/manageiq/blob/master/app/models/host.rb#L488) which is checking if vmotion is enabled.  All these other events are setup to allow control to prevent actions from happen and this is the only one that is checking if a feature is enabled.  That feature check does not seem like it needs to be prevented.

@chessbyte This `policy_prevented?` call was added in commit c86b5e9 back in August 2011 and it feels like a needless check.  Can you confirm?